Resources/config/adapter_factories.xml

```diff
@@ -62,6 +62,9 @@
         <service id="knp_gaufrette.adapter.factory.dropbox" class="Knp\Bundle\GaufretteBundle\DependencyInjection\Factory\DropboxAdapterFactory">
             <tag name="gaufrette.adapter.factory" />
         </service>
+        <service id="knp_gaufrette.adapter.factory.cloudinary" class="Zizoo\Bundle\CmsBundle\DependencyInjection\Factory\CloudinaryAdapterFactory">
```

Comment:
It is a bundle of external to the project I do not think that this service should appear in this project